test/document_stores/test_in_memory.py

```diff
@@ -74,7 +74,7 @@ def test_write_documents(self, document_store):
             document_store.write_documents(docs)
 
     def test_bm25_retrieval(self, document_store: InMemoryDocumentStore):
```

Comment:
If we initialize a new `InMemoryDocumentStore`, we can remove the `document_store` fixture here.

Reply:
Removed the initializer instead as the fixture is used in other tests.